client/src/utilities/date-time-display.test.ts

```diff
@@ -0,0 +1,43 @@
+import { describe, it, expect, vi } from 'vitest';
+import { dateDisplay, timeDisplay, durationDisplay } from './date-time-display';
+
+const mockToday = new Date(2024, 11, 20);
+Date.now = vi.fn(() => mockToday.getTime());
```

Comment:
Are you overwriting the current date here? 🤯
Alternatively, you could do something like this:
`expect(dateDisplay(Date.now()).toBe('Today');`
`expect(dateDisplay(Date.now()+(1000*60*60*24)).toBe('Tomorrow');`
But hey, if it works why change it.